packages/interface-connection/src/index.ts

```diff
@@ -180,11 +180,6 @@ export interface Connection {
   remotePeer: PeerId
   tags: string[]
   streams: Stream[]
-
-  newStream: (multicodecs: string | string[], options?: NewStreamOptions) => Promise<Stream>
-  addStream: (stream: Stream) => void
-  removeStream: (id: string) => void
-  close: () => Promise<void>
```

Comment:
only the `addStream` and `removeStream` methods should be deleted from this interface.